src/polytopes/chain.jl

```diff
@@ -4,59 +4,31 @@
 
 """
     Chain(p1, p2, ..., pn)
+    Chain(points::AbstractVector)
 
-A polygonal chain from a sequence of points `p1`, `p2`, ..., `pn`.
+A polygonal chain from a sequence of points `p1`, `p2`, ..., `pn`. If `points`
+is of type `CircularVector`, the chain is considered closed.
 See https://en.wikipedia.org/wiki/Polygonal_chain.
-"""
-struct Chain{Dim,T,V<:AbstractVector{Point{Dim,T}}} <: Polytope{1,Dim,T}
-  vertices::V
-end
-
-# specialize Polytope outer constructor to use standard vector
-Chain(vertices::Vararg{P}) where {P<:Point} = Chain(collect(vertices))
-
-Chain(vertices::CircularVector) = Chain([collect(vertices); first(vertices)])
-
-"""
-    npoints(chain)
-
-Return the total number of points used to represent the
-chain, no matter if it is closed or open.
-
-See also [`nvertices`](@ref).
 
 ### Notes
 
-This function is provided for IO purposes. Most algorithms
-should be written in terms of `nvertices` and `vertices`
-as they are consistent with each other.
+- `Ring` is a type alias for closed chains.
```

Comment:
```suggestion
- A `Ring` is a type alias for closed `Chain`.
```

Reply:
I’ve made the changes to docstrings more minimal in the newer version – feel free to change further.